Jenkinsfile

```diff
@@ -8,29 +11,20 @@ properties([
 node("primary") {
   echo "Clean workspace"
   cleanWs()
-  
+
   stage ("Checkout SCM") {
-    checkout scm
-    sh "mkdir -p jenkins-plumbing"
-    dir ("jenkins-plumbing") {
-      git([branch: "master", url:'ssh://git@stash:7999/van/jenkins-plumbing.git', credentialsId: '8aa93893-84cc-45fc-a029-a42f21197bb3'])
-    }
+    checkout localBranch(scm)
   }
 
-  def extExec = load("jenkins-plumbing/exec.groovy")
-  def extBedrock = load("jenkins-plumbing/bedrock-tests.groovy")
-  def extNpmInstall = load("jenkins-plumbing/npm-install.groovy")
-
   stage("Building") {
-    extNpmInstall()
-    extExec("yarn run build")
+    npmInstall()
```

Comment:
Should this have been called `yarnInstall` in walugi instead? as we aren't using npm anymore (or at least not at the moment) so this could be a bit misleading.

Reply:
Yeah, good idea. I'll address it in ARCH-110